scripts/sync_host_files.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+import json
+import os
+import subprocess
+
+
+def main():
```

Comment:
FYI @diox we discussed this recently. This logic is how we can ensure that the state of the container and host are synced even with mounts and regardless of mode, because on production images we reinstall deps fresh, build locales and assets essentially reflecting the state of the production image (those exact commands are run in the image build)